### DISCUSSION TITLE
Impromptu Supabase / Dev meetup in Taipei, Taiwan on Mon Jan 30th 2023

### DISCUSSION THREAD

**Discussion by thorwebdev** [supabase/supabase#12013] [Polls]
Any (supabase) devs in Taipei up for a spontaneous meetup tmr? Could go to WeWork or happy to hear suggestions for a casual cafe or lunch spot!

@JasonChiu-dev @fergusmeiklejohn @Yukaii @xJkit @ ss8651twtw @edisonlee55 @ljcucc @rlshepherd @erozak @dada878 @browny @mioyu512 @samuel22gj @davidho0403

**Comment by thorwebdev** [comment]
Thanks for voting. Let’s meet at 5pm at

掌門精釀啤酒-台北華山文創店 Zhangmen Taipei Huashan
+886 2 2322 3066
https://maps.app.goo.gl/K6xrPuVmVrCpF92z5?g_st=ic

Looking forward to it 😊

**Comment by Ropsten80** [comment]
hi
